admin/qtx_admin_taxonomy.php

```diff
@@ -166,6 +166,9 @@ function qtranxf_term_sanitize_name_unslashed( $term, $taxonomy = null ) {
  */
 function qtranxf_term_sanitize_name_db( $term, $taxonomy = null ) {
     global $q_config;
+
+    if ( ! qtranxf_isMultilingual( $term ) ) return $term;
+
```

Comment:
Yet it's unclear what to do with the slashes.
Even more surprising, if the term is not multilingual, the value returned by `qtranxf_term_sanitize_name_unslashed` is already a simple string (!), no ML tags. The reason is because it takes the default language anyway, even if the term is ML.
But... this change makes it work for some _other reason_, which is very convoluted... So, I don't understand yet how this works.
If we could simplify and remove all this mess of sanitization it would be nice. But there must (?) have been some reason for doing all of this.

Reply:
>  the value returned by qtranxf_term_sanitize_name_unslashed is already a simple string 

Point is not the value returned to the filter, but the value set for `$q_config['terms_sanitized'][ $term ]` in that function, which is taken from the previous translations from options and used later in `qtranxf_term_set_translation` to be written again to options.
Hence for this scenario if term name is not multilingual call to `qtranxf_term_sanitize_name_unslashed` is avoided and `$q_config['terms_sanitized'][ $term ]` is not set (so it is removed from translations in options later).

Possible explanation for current behaviour of `qtranxf_term_sanitize_name_unslashed` (which may be needed in other scenarios not affected by this commit) here: https://github.com/qtranslate/qtranslate-xt/issues/1215#issuecomment-1279713113

Reply:
Yes, got it. I was unsure if the calls for escaping with slashes were needed. But I tried with strings containing single quotes, double quotes and it works for all the cases (different values or all the same for each language).
Seems good now.